code/modules/urist/items/vgitems.dm

```diff
@@ -80,3 +80,73 @@
 	w_class = ITEM_SIZE_SMALL
 	obj_flags = OBJ_FLAG_CONDUCTIBLE
 	insultmsg = list("HONK!", "HELP, SHITCURITY!", "HELP MAINT", "SHITSEC", "I AM IN A CONSTANT STATE OF SUFFERING, EACH EON SPENT ON THIS PLANE OF EXISTENCE ECHOES ONLY PURE PAIN TO MY MORTAL COIL. GOD HAS DIED, AND HE HAS ONLY LEFT ME SUFFERING. ", "MY FUNNY BONE!")
+
+//lottery scratchers - west3436 of /vg/station13
+/obj/item/toy/lotto_ticket
+	name = "scratch-off lotto ticket"
+	desc = "A scratch-off lotto ticket."
+	icon = 'icons/urist/items/tgitems.dmi' //i see no reason to change this
+	w_class = ITEM_SIZE_TINY
+	var/revealed = FALSE
+	var/iswinner = FALSE
+	var/prize_multiplier
+	var/winnings = 0
+	var/list/prizelist = list(100000,50000,10000,5000,1000,500,250,100,50,20,10,5,4,3,2,1)
+	var/list/problist = list(0.0001, 0.0002, 0.001, 0.002, 0.01, 0.02, 0.04, 0.2, 1, 2.5, 5, 10, 12.5, 17, 20, 25)
+
+/obj/item/toy/lotto_ticket/New()
```

Comment:
I'd use Initialize() here, instead of New(), it's preferred to use over New()

Reply:
ok